cygnus-arcgis/pom.xml

```diff
@@ -0,0 +1,157 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
Looking to how the contribution is structured (cygnus-argis/ directory in the root) I wonder if this means a kind of "independent" Cygnus implementation with ArcGis support (as a sibling of Cygnus Twitter or Cygnus NGSI) or a sink for regular Cygnus (I mean, Cygnus NGSI) to be used along with all the other sinks.

As far as I understand, tha old implementation in PR https://github.com/telefonicaid/fiware-cygnus/pull/1683 was in that later way (a sink).

Reply:
Structure changed to fit into cygnus-common and cygnus-ngsi projects. c8d4419